src/score.py

```diff
@@ -77,4 +78,89 @@ def compute_weighted_acc(da_fc, da_true):
     )
     return acc
 
+def compute_weighted_meanspread(prediction,mean_dims=xr.ALL_DIMS):
```

Comment:
Could you adapt your scripts to use the same variable names as my other functions? Having them consistent will make this a lot less confusing in the future. So e.g. `prediction` should be `da_fc`